kazoo/recipe/lock.py

```diff
@@ -15,16 +15,9 @@
 
 """
 import re
-import sys
-
-try:
-    from time import monotonic as now
-except ImportError:
-    from time import time as now
+from time import monotonic as now
```

Comment:
Why the rename? Why not `import monotonic` and use `monotonic` at the callsite? The original rename was because of the try/catch import, but here that's no longer relevant...

Reply:
I would further say that I much prefer importing modules instead of classes or functions (kind of like one would use namespaces in C++).

This is much more manageable because you avoid any name collision and it is very clear where each symbol comes from.

``` python
import time

def foo():
    tick = time.monotonic()

```